website/docs/MocksAndSpies.md

```diff
@@ -54,36 +37,12 @@ Once you have defined a mock you can define custom responses for it. Those custo
 
 In order to mock API requests where you expect a JSON response all you need to do is to call `respond` on the mock object with an arbitrary object you want to return, e.g.:
 
-<!--DOCUSAURUS_CODE_TABS-->
-<!--Sync-->
-```js
-const mock = browser.mock('https://todo-backend-express-knex.herokuapp.com/', {
-    method: 'get'
-})
-
-mock.respond([{
-    title: 'Injected (non) completed Todo',
-    order: null,
-    completed: false
-}, {
-    title: 'Injected completed Todo',
-    order: null,
-    completed: true
-}])
-
-browser.url('https://todobackend.com/client/index.html?https://todo-backend-express-knex.herokuapp.com/')
-
-$('#todo-list li').waitForExist()
-console.log($$('#todo-list li').map(el => el.getText()))
-// outputs: "[ 'Injected (non) completed Todo', 'Injected completed Todo' ]"
-```
-<!--Async-->
 ```js
 const mock = await browser.mock('https://todo-backend-express-knex.herokuapp.com/', {
     method: 'get'
 })
 
-await mock.respond([{
+mock.respond([{
```

Comment:
mock.respond doesn't need the await here?

Reply:
nope, respond is a synchronous operation